### PR TITLE
Some additional clean-up

### PR DESCRIPTION
**Pull request by dmfalke**
Move active viz to realm of map type config, and clean up some labels and ids.

### CODE REVIEW

File: packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx [hunk 1]
```diff
@@ -580,12 +592,12 @@ function MapAnalysisImpl(props: ImplProps) {
...
         {
           type: 'subheading',
-          labelText: 'Single Variable Maps',
+          labelText: MapSideNavItemLabels.SingleVariableMaps,
           children: [
             {
               type: 'item',
               // concatenating the parent and subMenu labels creates a unique ID
-              id: MapSideNavItemLabels.ConfigureMap + MarkerTypeLabels.pie,
+              id: 'single-variable-pie',
```
Comment (jernestmyers) [inline packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx:600]: nit: we can remove the preceding comment re: concatenating labels

File: packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx [hunk 2]
```diff
@@ -684,7 +698,7 @@ function MapAnalysisImpl(props: ImplProps) {
             {
               type: 'item',
               // concatenating the parent and subMenu labels creates a unique ID
-              id: MapSideNavItemLabels.ConfigureMap + MarkerTypeLabels.barplot,
+              id: 'single-variable-bar',
```
Comment (jernestmyers) [inline packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx:701]: nit: same as above; comment can be removed

**Review by jernestmyers — APPROVED** [review]
LGTM 💪

File: packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx (hunk 1 above)
Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx:600]: Oh my, of course!

File: packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx (hunk 2 above)
Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx:701]: Yes!

**Pull request merged**